notebooks/official/automl/automl_tabular_on_vertex_pipelines.ipynb

```diff
@@ -0,0 +1,1083 @@
+{
```

Comment:
<p>"archiecture": spelling error</p><p> </p><p>I'm thinking it's cleaner to have one notebook to:</p><p>- demonstrate how to limit the search space</p><p>- demonstrate how to change machine types</p><p>- save tuning result</p><p> </p><p>And a second notebook to:</p><p>- load tuning result</p><p>- skip architecture search</p><p> </p><p>This way, we have one workflow per notebook. I am assuming here that the we can go through the notebook initialization steps very quickly or skip them altogether. If that's not the case, I would withdraw my suggestion here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/694/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Reply:
Thanks, Max. I fixed the typo. I think we can follow up later on the renaming.